ChatDemo/ViewController.swift

```diff
@@ -15,12 +15,6 @@ import StreamChat
 /// Use ChannelsViewController as the parent view controller.
 class ViewController: ChannelsViewController {
     required init?(coder: NSCoder) {
```

Comment:
We don't need this init anymore, tutorial also doesn't have this

Reply:
Done 👍